backbone.d3view.js

```diff
@@ -75,31 +75,54 @@
 
     // `delegate` supports two- and three-arg forms. The `selector` is optional.
     delegate: function(eventName, selector, listener) {
-      var el;
+      var el = this.el;
 
-      if (typeof selector === 'string') {
-        el = selector === '' ? this.d3el : this.d3el.selectAll(selector);
-      } else {
-        el = this.d3el;
+      if (listener === undefined) {
         listener = selector;
         selector = null;
       }
 
-      // d3 needs `uniqueId` to delegate more than one listener per event type.
-      var namespace = '.' + uniqueId++;
-
       var map = _eventsMap[this.cid] || (_eventsMap[this.cid] = {}),
           handlers = map[eventName] || (map[eventName] = []);
 
-      handlers.push({selector: selector, listener: listener, namespace: namespace});
-
-      // The `event` object is stored in `d3.event` but Backbone expects it as
-      // the first argument to the listener.
-      el.on(eventName + namespace, function() {
-        var args = slice.call(arguments);
-        args.unshift(d3.event);
-        listener.apply(this, args);
-      });
+      handlers.push({selector: selector, listener: listener});
+
+      var ElementProto = (typeof Element !== 'undefined' && Element.prototype) || {};
```

Comment:
pull this into the outer closure, along with the rest of the helpers. no need to recreate this on each call.